lib/Conversion/TorchToTosa/TorchToTosa.cpp

```diff
@@ -2645,6 +2645,24 @@ LogicalResult ConvertAtenOp<AtenViewOp>::matchAndRewrite(
     return rewriter.notifyMatchFailure(op,
                                        "size must consist of Scalar constants");
 
+  // # the size -1 is inferred from other dimensions
```

Comment:
You need to first make sure there is at most one `-1` in the list and return a `notifyMatchFailure` if that is not the case

Reply:
Done